privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -157,6 +161,33 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
         val cosmetic: Boolean? = false,
     )
 
+    data class RemoteFeatureSettingsViewState(
+        val primaryScreen: PrimaryScreenSettings = PrimaryScreenSettings()
+    ) {
+        companion object {
+            fun fromSettings(settings: PrivacyDashboardRemoteFeature): RemoteFeatureSettingsViewState {
+                return RemoteFeatureSettingsViewState(
+                    primaryScreen = PrimaryScreenSettings(
+                        layout = if (settings.protectionToggleHighlightActive().isEnabled()) {
+                            LayoutType.HIGHLIGHTED_PROTECTIONS_TOGGLE.value
+                        } else {
+                            LayoutType.DEFAULT.value
+                        }
+                    )
+                )
+            }
+        }
+    }
+
+    enum class LayoutType(val value: String) {
+        DEFAULT("default"),
+        HIGHLIGHTED_PROTECTIONS_TOGGLE("highlighted-protections-toggle");
+    }
+
+    data class PrimaryScreenSettings(
+        val layout: String = LayoutType.DEFAULT.value
+    )
+
```

Comment:
The dashboard just has an alternative layout - it knows nothing about experiments etc. Its the job of this view model to determine if we're in an experiment group and then apply the correct settings each time the dashboard is open

Reply:
The string values here match the documented strings in the updated Privacy Dashboard